templates/case-studies/index.html

```diff
@@ -0,0 +1,72 @@
+{% extends "case-studies/base_case-studies.html" %}
+
+{% block title %}Case Studies{% endblock %}
+
+{% block content %}
+  <!--This index page will be using macros to build an example case study -->
+  <!-- Hero section-->
+  {% import "case-studies/templates/_macros-hero.jinja" as hero_template %}
+  {{ hero_template.hero(title="Canonical deploys infrastructure solutions and managed IT services for critical space mission operations",
+                        subtitle="Hero description",
+                        logo="https://assets.ubuntu.com/v1/5b46609e-layer1.png",
+                        logo_alt="ESM logo",
+                        logo_width="107",
+                        logo_height="39")
+  }}
+  
+  <!-- Text + list section -->
+  {% import "case-studies/templates/_macros-text-list.jinja" as text_list_template %}
+  {{ text_list_template.text_list(
+      title="About the European Space Agency (ESA)",
+      list_items=[
+        "Founded in 1975, ESA is a 22-member intergovernmental space exploration organisation.",
+        "The European Space Operations Centre (ESOC), one of ESA’s premises in Germany, is home to the engineering teams that control spacecraft in orbit, manage the global tracking station network, and design and build the systems on the ground that support missions in space.",
+        "ESOC is responsible for a wide range of IT infrastructure and systems to help fly these unique missions, having been responsible for flying more than 80 custom-built satellites since it was established."
+      ]
+    ) 
+  }}
+
+  <!-- Highlights section -->
+  {% import "case-studies/templates/_macros-highlights.jinja" as highlights_template %}
+  {{ highlights_template.highlights(
+      list_items=[
+        "Architecture, design, and deployment of Canonical’s distributions of Ceph and Kubernetes.",
+        "Timely updates and 24/7 monitoring of missioncritical equipment.",
+        "ESA has operated more than 87 missions from ESOC, ranging from understanding our own planet to exploring our solar system and beyond to help advance scientific knowledge.",
+        "ESOC is responsible for a wide range of IT infrastructure and systems to help fly these unique missions, having been responsible for flying more than 80 custom-built satellites since it was established."
+      ]
+    ) 
+  }}
+
+  <!-- Text section -->
+  {% import "case-studies/templates/_macros-text.jinja" as text_template %}
+  {{ text_template.text(
+      title="You can’t explore space without infrastructure.",
+      description_list=[
```

Comment:
In general, we decided to reserve parameters to arguments that don't contain free HTML.

While it does look, like the strings you pass here are not HTML, I guess it's not uncommon to have some HTML in there? Links, etc?

And passing an array of strings to turn into paragraphs may be also hard to maintain (and format in the file).

So in cases like this, we would pass such block of text a "slot".
Not sure if you've been on last dev catchup when I demoed it, we also have some examples of this in Vanilla.

But this would be something like:

```
{%- call(slot) text_template.text(title="You can't explore…") -%}
   {%- if slot == "description" -%}
      <p>It's not just expert personell…</p>
      <p>ESA decided…</p>
   {%- endif -%}
{%- endcall -%}
```

Obviously it would require changes in the macro itself to accept the call with a child block.